drivers/azure/azureutil/azureutil.go

```diff
@@ -460,6 +460,21 @@ func (a AzureClient) removeOSDiskBlob(resourceGroup, vmName, vhdURL string) erro
 	if err != nil {
 		log.Debugf("Container remove happened: %v", ok)
 	}
+
+	cts, err := bs.GetBlobService().ListContainers(blobstorage.ListContainersParameters{})
+	if err != nil {
+		log.Debugf("Error listing containers.")
```

Comment:
Probably need to `return err` here as well, as the value of `cts` will likely be undefined.

Reply:
@kleinsasserm You're correct in that it will not cause a crash, but "falling through" here will cause the next condition `len(cts.Containers) == 0` to always be true, and could cause unwanted data loss if containers are in fact present. Also, returning `err` here will allow us to report the exact error that occurred downstream.

Reply:
Omg. Now I see, this is now embarrassing. 😊 I will try to do it the correct way in the next days. Thx @shin- for showing me my fallacy!